src/test/java/io/zold/api/WalletsInTest.java

```diff
@@ -36,6 +41,11 @@
  * Test case for {@link WalletsIn}.
  *
  * @since 0.1
+ * @todo #12:30min The name of the wallets in the test
```

Comment:
@victornoel the naming of the test wallets are OK for now.

> a test should be added to  ensure WalletsIn.create does overwrite and existing wallet.

You mean *does **not** overwrite **an** existing wallet*?

Reply:
@llorllale yes, good catch :)

Reply:
@llorllale I don't understand, do you want me to remove the whole `@todo` or that I just leave a small one about testing not overwriting files? That's seem too small for a task and it will come naturally in the future.

Reply:
@victornoel well, you can remove the `@todo` and leave a stubbed test in `WalletsInTest`

Reply:
@victornoel I still think it's a valid test. All I think you need to do is add the skeleton of a test that would check this (make sure you leave a `@todo` for it).

Reply:
@llorllale alright